src/server.spec.ts

```diff
@@ -236,6 +236,28 @@ describe("JSONRPCServer", () => {
     });
   });
 
+  describe("receiving a request to an unknown method with default set", () => {
```

Comment:
To connect `describe` and `it` descriptions:
```suggestion
  describe("receiving a request to an unknown method with a default method that responds null", () => {
```